src/desktop/apps/partner2/client/overview_layout_factory.coffee

```diff
@@ -205,3 +214,21 @@ galleryThreeHero = (partner, profile) ->
       when 'Artworks' then _.pick options, 'partner'
       else _.pick options, 'partner', 'maxNumberOfShows'
   )
+
+#
+# Sections for gallery_four layout.
+#
+galleryFourHero = (partner, profile) ->
```

Comment:
If this is identical to existing Premium partners, we can also just do
```
galleryFourHero = galleryThreeHero
```

Reply:
That's an amazing idea. I got rid of `galleryFourHero` altogether and directly used `galleryThreeHero` above.